doc/Makefile

```diff
@@ -5,10 +5,10 @@ include Makefile.sphinx
 
 CHPL2RST     = ../util/run-in-venv.bash ./util/chpl2rst.py
 
-CHPL_DOC_PRIMERS_DIR = primers/primers
+CHPL_DOC_PRIMERS_DIR = $(SOURCEDIR)/primers/primers
```

Comment:
I believe this was originally put into a sub-dir so that we could easily blow the generated primers away on a `make clobber` without protecting the top-level `index.rst`, and it also mirrors `examples/hellos`.

I think we can eliminate the sub-directory now that we copy `index.rst` files from `meta/` into generated directories. Would you like to see that in this PR?